Configurations/gentemplate.pm

```diff
@@ -362,6 +362,8 @@ sub dolib {
         my @objs = ();
         my @foreign_objs = ();
         my @deps = ();
+        my @incs = ();
+        my @defs = ();
```

Comment:
Regarding all modifications in this file (to `dolib`, `domodule` and `dobin`), I wonder where they come from...  the make zero sense to me, as all three functions handling linking object files and libraries into shared libraries, modules and executables, respectively.  When linking, header file inclusions and C macro definitions are irrelevant, so it makes no sense having these variables in those functions...